src/segmantic/seg/monai_unet.py

```diff
@@ -137,14 +166,15 @@ def default_augmentation(self, keys: List[str]):
             xforms.append(
                 RandZoomd(keys=keys, prob=0.2, min_zoom=0.8, max_zoom=1.3, mode=mode)
             )
-
+        # Set probability of background label being chosen as corp center to 0 and rest to 1.
         xforms += [
             RandCropByLabelClassesd(
                 keys=keys,
                 label_key="label",
                 spatial_size=self.spatial_size,
                 num_classes=self.num_classes,
-                num_samples=4,
+                num_samples=self.num_samples,
+                ratios=[0 if x == 0 else 1 for x in range(len(self.num_classes))],
```

Comment:
`len(self.num_classes)` is 1, I guess you meant `range(self.num_classes)`

Reply:
Yes, indeed^^. Thank you